doc/classes/EditorSettings.xml

```diff
@@ -688,6 +688,9 @@
 		<member name="interface/editor/project_manager_screen" type="int" setter="" getter="">
 			The preferred monitor to display the project manager.
 		</member>
+		<member name="interface/editor/remember_window_size_and_position" type="bool" setter="" getter="">
+			If [code]true[/code], the editor window will remember its size, position and which screen it was display on across restarts.
```

Comment:
```suggestion
			If [code]true[/code], the editor window will remember its size, position, and which screen it was display on across restarts.
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).